util/get_fisher_data.py

```diff
@@ -11,6 +12,7 @@
 locales = ['de', 'en', 'fr', 'ja']
 tackle_id = 30
 base = 'https://xivapi.com/'
+fishTrackerBase = 'https://raw.githubusercontent.com/icykoneko/ff14-fish-tracker-app/master/private/fishData.yaml'
```

Comment:
This seems like a lot of dependencies to get tug data.  Is the tug data in Saint Coinach or xivapi such that cactbot could look it up itself?  Or is this just user data that the yaml data contains?

Reply:
As far a s I’m aware neither of those contains the tug data- I don’t think the game client knows at all, since it simply plays whatever animation the server sends. The only other way to know this is via network logs, but the relevant packet for this is only listed under 252|... so I don’t know if that can be used.

The tug data is manually tracked (in either the spreadsheets or the yml file).

Reply:
Other information would be great if you wanted to add it!